examples/decoding/plot_ssd_spatial_filters.py

```diff
@@ -0,0 +1,142 @@
+"""
+===========================================================
+Compute Sepctro-Spatial Decomposition (SDD) spatial filters
+===========================================================
+In this example, we will compute spatial filters for retaining
+oscillatory brain activity and down-weighting 1/f background signals
+as proposed by :footcite:`NikulinEtAl2011`.
+The idea is to learn spatial filters that separate oscillatory dynamics
+from surrounding non-oscillatory noise based on the covariance in the
+frequency band of interest and the noise covariance absed on surrounding
+frequencies.
+References
+----------
+.. footbibliography::
+"""
+# Author: Denis A. Engemann <denis.engemann@gmail.com>
+#         Victoria Peterson <victoriapeterson09@gmail.com>
+# License: BSD (3-clause)
+
+import matplotlib.pyplot as plt
+import mne
+from mne import Epochs
+from mne.datasets.fieldtrip_cmc import data_path
+from mne.decoding import SSD
+from mne.utils import _time_mask
+
+
+def freq_mask(freqs, fmin, fmax):
+    """Convenience function to select frequencies."""
+    return _time_mask(freqs, fmin, fmax)
+
+
+# Define parameters
+fname = data_path() + '/SubjectCMC.ds'
+raw = mne.io.read_raw_ctf(fname)
+raw.crop(50., 110.).load_data()  # crop for memory purposes
+raw.resample(sfreq=250)
+
+picks_raw = mne.pick_types(
+    raw.info, meg=True, eeg=False, ref_meg=False)
+raw.pick(picks_raw)
```

Comment:
```suggestion
raw.pick_types(meg=True, eeg=False, ref_meg=False)
```
or directly you can do